src/components/local/transaction/PrivateTradesB.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import styled from 'styled-components';
+import ReactTable from 'react-table';
+
+export default class PrivateTradesB extends React.Component{
+
+    constructor(props) {
+        super(props);
+
+    }
+
+    state = {
```

Comment:
the state should be inside the constructor

Reply:
I fixed it